test/basic.js

```diff
@@ -1,76 +1,73 @@
-'use strict';
+import assert from "power-assert";
+import * as ShioriJK from "shiorijk";
+import {ShioriTransaction} from "../src/lib/shiori_transaction";
 
-if (typeof require !== 'undefined') {
-  var assert = require('power-assert');
-  var ShioriJK = require('shiorijk');
-  const shiori_transaction = require('../src/lib/shiori_transaction');
-  var ShioriTransaction = shiori_transaction.ShioriTransaction;
-}
-
-
-describe('ShioriTransaction', () => {
-  let request3, request2, response3, response2;
+describe("ShioriTransaction", () => {
+  let request3;
```

Comment:
__[jshint]__
*  'let' is available in ES6 (use 'esversion

<!-- Comment by [SideCI](https://sideci.com) -->